### PR TITLE
feat: 196 add endpoint to populate fixed bot messages

### PR DESCRIPTION
**Pull request by giobarbierii**
Created endpoint to populate fixed bot messages in DB

### CODE REVIEW

File: backend/src/test/kotlin/code/nebula/cipherquest/service/FixedBotMessageServiceTest.kt [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.mockito.Mockito
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest
+import org.springframework.boot.test.mock.mockito.MockBean
+import org.springframework.test.context.ActiveProfiles
+
+@WebMvcTest(FixedBotMessageService::class)
+@AutoConfigureMockMvc
```
Comment (mangialavorisara) [inline backend/src/test/kotlin/code/nebula/cipherquest/service/FixedBotMessageServiceTest.kt:16]: Should we replace these annotations with `@ExtendWith(MockitoExtension::class)`?

File: backend/src/test/kotlin/code/nebula/cipherquest/service/FixedBotMessageServiceTest.kt [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+@ActiveProfiles("test")
+class FixedBotMessageServiceTest {
+    @MockBean
```
Comment (mangialavorisara) [inline backend/src/test/kotlin/code/nebula/cipherquest/service/FixedBotMessageServiceTest.kt:19]: Should we use Mockito's `@Mock` instead?

File: backend/src/test/kotlin/code/nebula/cipherquest/service/FixedBotMessageServiceTest.kt [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+import org.mockito.Mockito
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest
+import org.springframework.boot.test.mock.mockito.MockBean
+import org.springframework.test.context.ActiveProfiles
+
+@WebMvcTest(FixedBotMessageService::class)
+@AutoConfigureMockMvc
+@ActiveProfiles("test")
+class FixedBotMessageServiceTest {
```
Comment (mangialavorisara) [inline backend/src/test/kotlin/code/nebula/cipherquest/service/FixedBotMessageServiceTest.kt:18]: Should we add negative test cases?

File: backend/src/main/resources/http/client.http [hunk 4]
```diff
@@ -36,6 +36,37 @@ Content-Type: application/json
...
+    {
+        "type": "GAME_OVER",
+        "content": "Sorry, you've lost :("
+    },
+    {
+        "type": "CHEAT_DETECT",
+        "content": "Cheater!!"
+    }
+]
+}
```
Comment (mangialavorisara) [inline backend/src/main/resources/http/client.http:68]: Can we fix the indentation?

File: backend/src/test/kotlin/code/nebula/cipherquest/controller/FixedBotMessageControllerTest.kt [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+import org.springframework.test.web.servlet.MockMvc
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers.status
+import org.testcontainers.shaded.com.fasterxml.jackson.databind.ObjectMapper
+
+@WebMvcTest(FixedMessageController::class)
+@AutoConfigureMockMvc
+@ActiveProfiles("test")
+class FixedBotMessageControllerTest {
```
Comment (mangialavorisara) [inline backend/src/test/kotlin/code/nebula/cipherquest/controller/FixedBotMessageControllerTest.kt:24]: Should we also test negative cases? e.g., blank fields, empty message list
